regional_mom6/rotation.py

```diff
@@ -133,8 +133,8 @@ def mom6_angle_calculation_method(
     point: xr.DataArray,
 ) -> xr.DataArray:
     """
-    Calculate the angle of the grid point using the MOM6 method adapted from the
-    MOM6 code: https://github.com/mom-ocean/MOM6/blob/05d8cc395c1c3c04dd04885bf8dd6df50a86b862/src/initialization/MOM_shared_initialization.F90#L572-L587
+    Calculate the angle of the grid point's local constant-y direction compared to East-West direction
```

Comment:
"local constant-y direction" makes sense but is pretty confusing to me. I think it is more common to reference true north & grid north (happy if someone thinks differently). I would change to something like 
```suggestion
    Calculate the angle of the grid point's local constant-x direction compared to the true north direction
```
across the documentation. Especially since that's the way MOM6 does it.

Reply:
Yeah, that makes sense. That angle is the same as the local x from north-south right?